base/applications/rapps/lang/it-IT.rc

```diff
@@ -45,22 +45,6 @@ BEGIN
     END
 END
 
-IDR_APPLICATIONMENU MENU
-BEGIN
-    POPUP "popup"
-    BEGIN
-        MENUITEM "&Installa\tCtrl+Enter", ID_INSTALL
-        MENUITEM "&Rimuovi\tCtrl+Del", ID_UNINSTALL
-        MENUITEM "&Modifica", ID_MODIFY
-        MENUITEM SEPARATOR
-        MENUITEM "&Cancella dal registro", ID_REGREMOVE
-        MENUITEM SEPARATOR
-        MENUITEM "&Aggiorna\tF5", ID_REFRESH
-        MENUITEM SEPARATOR
-        MENUITEM "Aggiorna Da&tabase\tCtrl+F5", ID_RESETDB
-    END
-END
-
```

Comment:
Just a question: That cleanup part of your PR of removing IDR_APPLICATIONMENU in all rcs... Is that possible also without the rest of your PR?

Reply:
Yes, this removal is in a separate commit that can compile without the rest.